config/application.rb

```diff
@@ -23,6 +23,16 @@ class Application < Rails::Application
     config.generators.test_framework :rspec
     config.eager_load_paths += %W[#{config.root}/lib]
 
```

Comment:
I don't think this relate to this patch ?
why do we need CORS in authz component ?